woocommerce/wc-admin
# Render a spinner while woocommerce_setup_jetpack_opted_in is being loaded

Fixes #7239 

This PR preloads `woocommerce_setup_jetpack_opted_in` option to prevent Jetpack and WC Tax TOS text flickering.

The TOS text gets rendered first then we make an Ajax request to get the value of [tosAccepted](https://github.com/woocommerce/woocommerce-admin/blob/main/client/task-list/tasks/tax.js#L536). If the value returned is true, then `tosAccepted` becomes true and the text disappears. Since Ajax requests can take a second, the text might blink depending on the value returned. Preloading the option value fixes the problem.

### Screenshots

![Screen Shot 2021-06-29 at 2 36 50 PM](https://user-images.githubusercontent.com/4723145/123870682-8c6e0e80-d8e7-11eb-8c00-13d5db05b8a4.jpg)


### Detailed test instructions:

1. Go to OBW setup wizard.
1. Uncheck the "WooCommerce shipping" and "WooCommerce Tax" options at Free features step.
1. Complete the OBW setup.
1. Go to WooCommerce->Home.
1. Click on "Set up Tax" option on Task list.
1. TOS should not blink.
